### ISSUE TITLE
demo

### ISSUE THREAD

**Issue by buendia** [HenrikJoreteg/Capsule#3]
please add a _wroking_ demo. pages and pages of documentation with no demo is not very helpful.

**Comment by HenrikJoreteg** [comment]
Yes, i'll post a demo when I get around to it. Unfortunately, working on open source projects is not very profitable. 

I'm building an app along side with this project and sharing what I can from the app in an attempt to give back to those that are interested. (Hence all the disclaimers in the readme)

Once I feel things are a bit more stable and we find the time, I'll shore up this lib with demos and better docs.

You're free to contribute :)
